lib/pages/home/drawer/home_drawer.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class HomeDrawer extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      child: Row(
+        children: <Widget>[
+          Expanded(
+            flex: 2,
+            child: _allGroups(),
+          ),
+          Expanded(
+            flex: 5,
+            child: _menus(),
+          )
+        ],
+      ),
+    );
+  }
+
+  Widget _menus() {
+    final List<String> _eventDetail = [
```

Comment:
特に理由なければ`_`つける必要なさそう
```suggestion
    final List<String> eventDetail = [
```

因みにSwiftではメソッド内の値にアクセス修飾子をつけることはできんかったはず


ここら辺は`_`付けてないけん一貫性が欲しいね
```Dart
  Widget _allGroups() {
    final List<Widget> groups = [
```

Reply:
androidもメソッドないでprivateはつけないし統一しよう！